oci/tests/integration/terraform/aws/main.tf

```diff
@@ -23,6 +23,12 @@ module "test_ecr" {
   tags = var.tags
 }
 
+resource "aws_ecrpublic_repository" "test_ecr_public" {
+  repository_name = local.name
+  tags            = var.tags
+  force_destroy   = true
+}
```

Comment:
Do we clean this up at the end of the CI run?

Reply:
Yes, on terraform destroy. But we may need to add code in the reaper, same for GCP WI pools and providers